src/vmcs/src/ept_intel_x64.cpp

```diff
@@ -82,19 +96,10 @@ ept_intel_x64::remove_page(integer_pointer gpa, integer_pointer bits)
             if (pt->empty())
             {
                 (*iter) = nullptr;
-
-                auto &&view = gsl::make_span(m_ept, ept::num_entries);
-                view.at(index) = 0;
+                entry.clear();
```

Comment:
you got me :)... hahaha jk, yeah I forgot that we stopped storing the entries in the m_epts. You can ignore this